### DISCUSSION TITLE
Option to disable the Test Request button

### DISCUSSION THREAD

**Discussion by facus26** [scalar/scalar#3005] [Ideas]
Hi, I love the project. 
I think it would be a good idea to add a way to disable the 'Test Request' button to prevent unnecessary requests.

**Comment by hanspagel** [comment]
Can you further elaborate why you think there are unnecessary requests?

**Reply by facus26** [reply]
Hi,
At my current job, we have the 'Test Request' button disabled because we don't have a testing environment to use as a proxy, and we don’t want traffic generated through the documentation. Having it enabled makes it very tempting to click.
Our solution for now was to implement ReDoc, which doesn’t have this button.

**Reply by hanspagel** [reply]
Got it! Submitted a PR: #3016
